modules/statistics/php/NDB_Form_statistics.class.inc

```diff
@@ -687,9 +719,9 @@ class NDB_Form_Statistics extends NDB_Form
         }
         $subprojList = substr($subprojList, 0, -1);
         if (!empty($subprojList)) {
```

Comment:
Instead, go for a construction like:

```php
$subproject_query = '';
if (!empty($subprojList)) {
    $subproject_query .= "AND s.SubprojectID IN ($subprojList)";
}
```

Basically no `else` statement. It's a bit cleaner and Dave will just **love** you for it.